weaviate/aliases/executor.py

```diff
@@ -122,3 +122,13 @@ def resp(res: Response) -> bool:
                 error="update aliases",
             ),
         )
+
+    def exists(self, *, alias_name: str) -> executor.Result[bool]:
+        """Check if an alias exists."""
+        self._connection._weaviate_version.check_is_at_least_1_32_0("alias")
+
+        try:
+            self.get(alias_name=alias_name)
+            return True
+        except Exception:
```

Comment:
I think we should not catch all exceptions, for example when the client cannot connect or is unauthenticated this can be a footgun (which Daniel found out a couple of weeks ago: https://github.com/weaviate/weaviate-python-client/issues/1799)